vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExporterController.java

```diff
@@ -127,7 +129,14 @@ private VariantExporterController(String dbName,
         query.setStudies(studies);
         evaWsClient = getChromosomeWsClient(dbName, evaProperties);
         regionFactory = new RegionFactory(windowSize, variantService);
-        exporter = new VariantExporter();
+
+        List<String> exclusions = query.getExclusions();
```

Comment:
The constructor is growing too much... what if we create a private method for getting excludeAnnotations?